THIS Event 2020/Abstract.tex

```diff
@@ -0,0 +1,18 @@
+\documentclass{article}
+
+\title{A game theoretic model of the behavioural gaming that takes place at the EMS - ED interface}
+\author{Michalis Panayides}
+
+\begin{document}
+
+\maketitle
+\begin{abstract}
+    Mathematical modelling can be utilised to model a vast range of scenarios applied to areas like healthcare, supermarkets and many more. Although, such methods are really powerful, they sometimes lack the possible insight that one may get by considering more qualitative approaches. The \textit{mother} of such approaches is considered by many to be ethnography. 
+
+    Ethnographic studies have been around since the \(19^{th}\) century and their main ideology is that a researcher that aims to study a specific social setting should acquire a deeper understanding of a culture's norms and values. The main technique that accompanies ethnography is participant observation, where the researcher participates in the social setting they wish to study and also records any observations made on it. Two additional concepts of ethnography are the ideas of emic models and etic models. An etic model is built based solely on the knowledge of the modeller and how they perceive the model. On the contrary an emic model takes into consideration the perspective of the individuals that belong in the social setting of interest. Ethnography suggests that an emic model should always be the default type of model used when studying any kind of social setting.
```

Comment:
```suggestion
    Ethnographic studies have been around since the \(19^{th}\) century and their guiding methodology is that a researcher that aims to study a specific social setting should acquire a deeper understanding of the cultural norms and values. The main technique that accompanies ethnography is participant observation, where the researcher participates in the social setting they wish to study recording their observations.
```